addons/message_update_v2.14/manifest.json

```diff
@@ -1,34 +1,41 @@
 {
   "api_version": "0.1",
-  "id": "message_update_v2.13",
-  "name": "Update to Mozilla VPN 2.13",
+  "id": "message_update_v2.14",
+  "name": "Update to Mozilla VPN 2.14",
   "type": "message",
   "conditions": {
-    "max_client_version": "2.12.9"
+    "max_client_version": "2.13.9"
   },
   "javascript": {
     "enable": "enable.js"
   },
   "message": {
     "date": 1671066000,
-    "id": "message_update_v2.13",
-    "title": "Update to Mozilla VPN 2.13",
+    "id": "message_update_v2.14",
+    "title": "Update to Mozilla VPN 2.14",
     "subtitle": "We’ve released an updated version of Mozilla VPN! Update to the latest version to get the newest features and bug fixes:",
     "badge": "new_update",
     "blocks": [
-      { "id": "c_1",
+      {
+        "id": "c_1",
         "type": "ulist",
         "content": [
-          { "id": "l_2",
-            "content": "When running a speed test, you’ll now be able to see upload as well as download speed." },
-          { "id": "l_3",
-            "content": "You can now see the IP address of the server you are connected to by tapping the info button." },
-          { "id": "l_4",
-            "content": "On Android, you can now turn the VPN on or off by using the quick actions in the notification bar." },
-            { "id": "l_5",
-            "content": "New multi-hop and app exclusion tutorials." },
-            { "id": "l_6",
-            "content": "We’ve also made bug fixes, UI adjustments and other performance improvements to make Mozilla VPN even better." }
+          {
+            "id": "l_1",
+            "content": "You can now choose to filter out malware, ads, and ad trackers, giving you more protection as you navigate the web. Find these features In “Privacy features” within the Settings menu."
```

Comment:
"In" doesn't need to be capitalized

Reply:
I am just a copy paster here. So, pinging @chrisrooneys :)

Reply:
Ok actually, let's go on a limb here because Chris is on PTO until after string freeze. I'll change this myself :pray:

Reply:
```suggestion
            "content": "You can now choose to filter out malware, ads, and ad trackers, giving you more protection as you navigate the web. Find these features in “Privacy features” within the Settings menu."
```